llvm/tools/llvm-mca/Views/BottleneckAnalysis.cpp

```diff
@@ -641,5 +641,51 @@ void BottleneckAnalysis::printView(raw_ostream &OS) const {
   printCriticalSequence(OS);
 }
 
+json::Value BottleneckAnalysis::toJSON() const {
+  if (!SeenStallCycles || !BPI.PressureIncreaseCycles) {
+    json::Object JO({{"PressureIncreaseCycles", 0}});
+    return JO;
+  }
+
```

Comment:
nit: would it be better to use `llvm::json::OStream` ([link](https://llvm.org/doxygen/classllvm_1_1json_1_1OStream.html#details)) here rather than storing serialized json objects/array?

Reply:
Apologies for the delay in replying. 

I'm afraid a re-factor of how MCA does json serialisation is beyond the resources I have been permmited to allocate to this task. This change does follow the conventions used for other json writing in MCA.